src/client/components/SignUpComponent/index.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import SignUpForm from '../SignUpForm';
+
+let url = 'http://localhost:8080';
+class SignUpComponent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      'status': 'being',
+      'error': '',
+      'pass': true,
+    };
+  }
+  submitHandler(ev) {
+    ev.preventDefault();
+    let userName = ev.target.elements.namedItem('username').value;
+    let email = ev.target.elements.namedItem('email').value;
+    let phone = ev.target.elements.namedItem('phone').value;
+    let fullName = ev.target.elements.namedItem('fullname').value;
+    let password = ev.target.elements.namedItem('password').value;
+    let obj = {
+      username: userName,
+      email: email,
+      phone: phone,
+      fullname: fullName,
+      password: password,
+    };
+    let jsonData = JSON.stringify(obj);
+    let xhr = new XMLHttpRequest();
+    xhr.addEventListener('readystatechange', function() {
+      if (xhr.readyState === XMLHttpRequest.DONE) {
```

Comment:
we should change back the state of the component, or in this case it will be loading ever after we clicked on the submit